www/assets/js/components/SearchFilterDrawer.tsx

```diff
@@ -72,25 +77,39 @@ export default function SearchFilterDrawer(props: Props) {
   ) : (
     <div className="controls-outer">
       <div className="controls">
-        <div onClick={openDrawer} className="filter-controls">
+        <div
+          onClick={openDrawer}
+          className="filter-controls"
+          role="button"
+          tabIndex={0}
+        >
           Filter
-          <i className="material-icons">arrow_drop_down</i>
+          <i className="material-icons" aria-hidden="true">
+            arrow_drop_down
+          </i>
         </div>
       </div>
       <div className="layout-buttons layout-buttons-mobile">
-        <button onClick={() => updateUI(null)} className="layout-button-left">
+        <button
+          onClick={() => updateUI(null)}
+          className="layout-button-left"
+          aria-label="search results with thumbnails"
+        >
```

Comment:
**Requested Change:** Add `role="button"`.

The default role for button elements is `"submit"`. =/

Reply:
yeah, I am definitely confusing it with `type` :) 

You can ignore that comment. _Though I guess maybe we should set `type=button`. _